libs/feature/map/src/lib/map-context/component/map-context.component.ts

```diff
@@ -25,19 +26,35 @@ export class MapContextComponent implements OnChanges {
   @Input() context: MapContextModel
   @Output() featureClicked = new EventEmitter<Feature<Geometry>[]>()
 
+  resizeObserver = new ResizeObserver(() => {
+    if (this.context.extent) {
+      this.map.updateSize()
+      this.context.view = this.utils.getViewFromExtent(
```

Comment:
I may missed something but why not using the `map.fit()` method ?
Has the map already a `View `or not BTW ?

Reply:
No, when accessing the map for the first time it has no `View` yet, so this prevents creating a default view and fitting on the extent afterwards. Apart from that, `fit()` should also work now with the `ResizeObserver`.